core/deployment/src/main/java/io/quarkus/deployment/util/FileUtil.java

```diff
@@ -78,14 +78,15 @@ public static byte[] readFileContents(InputStream inputStream) throws IOExceptio
      * @param windowsStylePath A path formatted in Windows-style, e.g. "C:\foo\bar".
      * @return A translated path accepted by Docker, e.g. "//c/foo/bar".
      */
-    public static String translateToVolumePath(String windowsStylePath) {
+    public static String translateToVolumePath(String windowsStylePath, boolean isPodman) {
         String translated = windowsStylePath.replace('\\', '/');
         Pattern p = Pattern.compile("^(\\w)(?:$|:(/)?(.*))");
         Matcher m = p.matcher(translated);
         if (m.matches()) {
             String slash = Optional.ofNullable(m.group(2)).orElse("/");
             String path = Optional.ofNullable(m.group(3)).orElse("");
-            return "//" + m.group(1).toLowerCase() + slash + path;
+            // Ad `/' and `//' see https://github.com/containers/podman/issues/14414
+            return (isPodman ? "/" : "//") + m.group(1).toLowerCase() + slash + path;
```

Comment:
This lgtm but its probably better if we switch everything over (docker and podman) to just / (or alternatively using the windows path directly which both support as well). Then we have consistent behavior.

Reply:
Feel free to reject my review if you disagree and I’ll merge anyway

Reply:
@n1hility Thx for the review. I changed only what I could actually test right now. That is WSL2 backed Podman. Various (possibly older) versions of Docker Desktop, that also varies by the backend being WSL2 or Hyper-V are not what I can test now. 

I would like this to be merged as it is so as it starts working for users now.

When/if I lay my hands on Docker Desktop for Windows I'll be glad to revisit this part of code.

Reply:
I'll assume this is resolved for now - we can improve it later.